src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import ClickAwayListener from '../../utilities/ClickAwayListener';
+import { Props } from './types';
+
+
+/**
+ * The Drawer component is used when you want to have content slide out from either the Top, Right, Bottom, or Left
+ */
+function Drawer({
+    anchorFrom,
+    children,
+    className,
+    isOpen = false,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [open, setOpen] = React.useState(isOpen);
+
+
+    React.useEffect(() => {
+        setOpen(isOpen);
+    }, [isOpen]);
```

Comment:
There is no reason to manage the isOpen state internally when it's coming through the props, so these lines are unnecessary